one_fm/one_fm/custom/job_applicant.json

```diff
@@ -4980,6 +5040,66 @@
    "unique": 0,
    "width": null
   },
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 0,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2020-11-30 21:46:48.303658",
+   "default": null,
+   "depends_on": null,
+   "description": null,
+   "docstatus": 0,
+   "dt": "Job Applicant",
+   "fetch_from": "job_title.designation",
+   "fetch_if_empty": 0,
+   "fieldname": "one_fm_designation",
```

Comment:
This custom field is removed one, since ERPNext core introduced this field in Job Applicant as designation